var/spack/repos/builtin/packages/duckdb/package.py

```diff
@@ -16,8 +16,106 @@ class Duckdb(CMakePackage):
 
     license("MIT")
     maintainers("glentner")
+    maintainers("teaguesterling")
 
     version("master", branch="master")
+    version("0.10.0", sha256="5a925b8607d00a97c1a3ffe6df05c0a62a4df063abd022ada82ac1e917792013")
     version("0.9.2", sha256="afff7bd925a98dc2af4039b8ab2159b0705cbf5e0ee05d97f7bb8dce5f880dc2")
+    version("0.9.1", sha256="37a43188d9354ce3ca101b2b118d867f5f76d04c3b83c09d86fd7508351a631b")
+    version("0.9.0", sha256="3dbf3326a831bf0797591572440e81a3d6d668f8e33a25ce04efae19afc3a23d")
     version("0.8.1", sha256="a0674f7e320dc7ebcf51990d7fc1c0e7f7b2c335c08f5953702b5285e6c30694")
+    version("0.8.0", sha256="df3b8e0b72bce38914f0fb1cd02235d8b616df9209beb14beb06bfbcaaf2e97f")
     version("0.7.1", sha256="67f840f861e5ffbe137d65a8543642d016f900b89dd035492d562ad11acf0e1e")
+
+    depends_on("python@3:")
```

Comment:
The python extension for duckdb needs version 3.7 minimum. I did not find a minimum version for duckdb without the extension, but I would be surprised if all of the included scripts would run with Python 3.0. I think this should be higher.
Since Spack needs minimum 3.6, it is probably safe to just bump this up.

Reply:
Agreed. Probably safest to aim for 3.7 without any further guidance from duckdb.